pkg/Microsoft.NETCore.Platforms/Microsoft.NETCore.Platforms.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <PackageVersion>$(PlatformPackageVersion)</PackageVersion>
+    <PackageVersion>$(MicrosoftNETCorePlatformsPackageVersion)</PackageVersion>
```

Comment:
@dagood @safern this change is going to conflict when we merge with the release/2.0.0 branch because we need to use a fixed version when producing this library in corefx as oppose to the incoming version from dependencies.props.

Reply:
Yup, we just changed this to be fixed to 2.0.1 for the 2.0.3 release.